contributor-docs/operations/tests/rfc2119-keyword-test-guide.md

```diff
@@ -0,0 +1,156 @@
+requirements could be marked in green, unfulfilled ones in red.
+* **SHOULD and SHOULD NOT**: These represent recommendations. While fulfilling these do
+not directly affect standard compliance, it is advised for future-proofing against
+potential standard evolutions. Non-compliance could be visualized in yellow, possibly
+with a valid reason for non-fulfillment.
+* **MAY**: This is more flexible and could be used to provide additional
+information or best practices. Their impact on compliance is not as direct as **MUST** or
+**SHOULD** categories. These can be marked as blue.
+
+## 3. Channels for Output in Test Scripts
+
+* Align the RFC2119 keywords with specific channels in test scripts:
+  * **MUST** and **MUST NOT** lead to the ERROR channel for failed tests
+  * **SHOULD** and **SHOULD NOT** lead to the WARNING channel, the test is still passed
+  * **MAY** to leads to the INFO channel, the test is still passed
+
+## 4. Compliance and Test Passing Criteria
+
+A test is considered to pass if and only if it doesn't produce any messages on the
+ERROR nor the CRITICAL channel. The presence of warnings or informational items from
+**SHOULD** and **MAY** categories do not directly impact the pass/fail status but is
+important for overall quality and compatibility with future standards.
+
+A test can have one of three results:
+
+* fail: if it has at least one message on ERROR
+* DNF (did not finish): if it has no message on ERROR, but at least one on CRITICAL
+* pass: otherwise
+
+## 5. Example: Standards Document using RFC2119 Keywords
+
+The following example of a fictitious document for a "Web Server Installation Standard"
+is intended to illustrate the use of RFC2119 keywords in connection with the creation of
+SCS-compliant standard documents.
+
+### Web Server Installation Standard
+
+#### Purpose
+
+This document provides a concise set of requirements for installing a secure web server.
+
+#### Requirements
+
+* Web Server Software: Apache HTTP Server 2.4.x or Nginx 1.18.x **MUST** be installed. No
+other web server software versions or types are permitted.
+
+* Operating System: The server **MUST** run on an OS that receives current security updates.
+Unsupported versions **MUST NOT** be used.
+
+* TLS Configuration: TLS 1.2 or higher **MUST** be enabled for all connections. SSL and
+earlier TLS versions **MUST NOT** be used.
+
+* Security Patches: The web server and OS **MUST** be kept up-to-date with security patches.
+Critical patches **SHOULD** be applied within 48 hours of release.
+
+* Firewall Setup: The firewall **SHOULD** be configured to allow only necessary ports,
+such as 80 (HTTP) and 443 (HTTPS). All unnecessary ports **SHOULD NOT** be open.
+
+* Backup Plan: Regular backups of the web server data **MAY** be performed, and it is
+recommended that backup integrity checks are conducted to ensure recoverability.
+
+#### Compliance
+
+Failure to comply with these requirements results in the web server being
```

Comment:
```suggestion
Failure to comply with these requirements results in the web server installation being
```

As I understand it, the example standard is about the installation (and configuration) of the webserver and its environment and not the webserver itself.

Reply:
Here I disagree, if the installation is not followed strictly and therefore not compliant, the webserver itself will be not compliant with these rules.